.github/workflows/high-impact-alert.yml

```diff
@@ -1,7 +1,7 @@
 name: High Impact Alert
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
Just to double check, I know you tested this: The payload will stay the same? `const { html_url, number, title } = context.payload.pull_request;`

Reply:
The webhook payload will be [pull_request](https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request) so self-answer: Yes

Reply:
I'd say those stay the same, yes, they still use the correct PR data

<img width="896" alt="Screenshot 2024-05-27 at 14 52 18" src="https://github.com/adobecom/milo/assets/11267498/a159382c-202a-47cb-921d-743f42720629">